policies/gitlab/organization.rego

```diff
@@ -81,3 +81,23 @@ default group_does_not_enforce_branch_protection_by_default := true
 group_does_not_enforce_branch_protection_by_default := false {
 	input.default_branch_protection > 0
 }
+
+# METADATA
+# scope: rule
+# title: Two-Factor Authentication Grace Period Should Not Be Longer Than One Week
+# description: The group allows more than one week for new members to add two-factor authentication. This time leaves your organization vulnerable to credential theft attacks whenever adding new members, especially in large developer groups.
```

Comment:
I would rewrite this sentence, it's kinda messy

Reply:
rephrased :)